src/gateway.rs

```diff
@@ -84,9 +84,30 @@ impl Gateway {
                     "ignoring send to client with unknown MAC: {:?}", mac
                 )
             }
-            Event::RawPacket(raw) => {
-                info!(logger, "ignoring raw packet {:?}", raw)
-            }
+            Event::RawPacket(raw) => match raw {
+                semtech_udp::Up::PushData(packet) => {
+                    if let Some(rxpks) = packet.data.rxpk {
+                        let mac = packet.gateway_mac;
+                        for rxpk in rxpks {
+                            info!(
+                                logger,
+                                "uplink from {}, {} MHz, {:?}, {}, snr = {},len = {}",
+                                mac,
+                                rxpk.get_frequency(),
```

Comment:
could you make this the rxpk Display trait?

Reply:
Yes, that sounds pretty reasonable. I'll make that change in `semtech-udp`